x-pack/legacy/plugins/canvas/public/components/border_resize_handle/border_resize_handle.scss

```diff
@@ -7,7 +7,7 @@
   width: 8px;
   margin-left: -4px;
   margin-top: -4px;
-  background-color: #fff;
-  border: 1px solid #666;
-  box-shadow: 0 2px 2px -1px rgba(153, 153, 153, 0.3), 0 1px 5px -2px rgba(153, 153, 153, 0.3);
+  background-color: $euiColorEmptyShade;
+  border: 1px solid $euiColorDarkShade;
+  box-shadow: 0 2px 2px -1px transparentize($euiColorMediumShade, .7), 0 1px 5px -2px transparentize($euiColorMediumShade, .7);
```

Comment:
Not 100% sure what this is used for, but a reminder we have a bunch of shadow mixins. They may or may not be a good sub.

https://github.com/elastic/eui/blob/master/src/global_styling/mixins/_shadow.scss